### PR TITLE
Web SDK 5.7.0 documentation update

### PR DESCRIPTION
**Pull request by ivosam3**
…K 5.7.0 release

## Related issues

Updated information for the release of Web SDK 5.7.0

- Jira issue link: https://adjustcom.atlassian.net/browse/ADAPP-18571?atlOrigin=eyJpIjoiMTYxNDM1ZjZlM2Q5NGQyNzkxYmYzMTAyMDlhNjdjNDMiLCJwIjoiaiJ9
- GitHub issue link:

## Changes

Updates privacy settings, device information, and attribution articles.  

## Required translations

Not sure how Web SDK should be translated

### CODE REVIEW

File: src/content/docs/en/sdk/web/features/privacy.mdx [hunk 1]
```diff
@@ -27,20 +27,91 @@ You can send the user's erasure request to Adjust by calling the `gdprForgetMe`
...
+
+<MinorVersion
+   changed="5.7.0"
+   link="https://github.com/adjust/web_sdk/releases/tag/v5.7.0"
+/>
+
+To disable third-party sharing for all users, call the disableThirdPartySharing method. When Adjust's servers receive this information, Adjust stops sharing the user's data with third parties. The Adjust SDK continues to work as expected.
 
-```js
 Adjust.disableThirdPartySharing();
```
Comment (YaraMatkova) [inline src/content/docs/en/sdk/web/features/privacy.mdx:84]: It doesn't look like a code block in [preview](https://dev-docs-bktiljsrq-product-content.vercel.app/en/sdk/web/features/privacy#disable-third-party-sharing), is it intentional?

File: src/content/docs/en/sdk/web/features/privacy.mdx [hunk 2]
```diff
@@ -27,20 +27,91 @@ You can send the user's erasure request to Adjust by calling the `gdprForgetMe`
...
+-  `key` (`string`): The option key.
+-  `value` (`string`): The option value.
+
+```ts
+const adjustThirdPartySharing = new Adjust.ThirdPartySharing(true);
+options.addGranularOption("PartnerA", "foo", "bar");
+Adjust.trackThirdPartySharing(options);
 ```
 
 ## Data residency {#data-residency}
```
Comment (YaraMatkova) [inline src/content/docs/en/sdk/web/features/privacy.mdx:117]: Would it be possible to mention that `dataResidency` property of `init` function is deprecated and should be replaced with `urlStartegy` with the appropriate configuration as value?

Migration example.
Old code:
```
Adjust.initSdk({
   appToken: "YOUR_APP_TOKEN",
   environment: "sandbox",
   dataResidency: "EU",
});
```

New code:
```
Adjust.initSdk({
   appToken: "YOUR_APP_TOKEN",
   environment: "sandbox",
   urlStrategy: {
      domains: ["eu.adjust.com"],
      useSubdomains: true,
      isDataResidency: true,
   },
});
```

File: src/content/docs/en/sdk/web/features/privacy.mdx [hunk 3]
```diff
@@ -74,27 +145,42 @@ Adjust.initSdk({
...
 
-</CodeBlock>
+-  The country in which Adjust stores your data (data residency).
+-  The endpoint to which the Adjust SDK sends traffic (URL strategy).
+
+This is useful if you're operating in a country with strict privacy requirements. When you set your URL strategy, Adjust stores data in the selected data residency region or sends traffic to the chosen domain.
+
+To set your country of data residency, call the `AdjustConfig.setUrlStrategy` method following parameters:
+
+-  `domains` (`any`): The country or countries of data residence, or the endpoints to which you want to send SDK traffic.
```
Comment (YaraMatkova) [inline src/content/docs/en/sdk/web/features/privacy.mdx:162]: A nitpick: `domains` property has `Array<string>` type. All type declarations are available here [Web SDK types](https://github.com/adjust/web_sdk/blob/master/dist/adjust-latest.d.ts)

File: src/content/docs/en/sdk/web/features/privacy.mdx [hunk 4]
```diff
@@ -74,27 +145,42 @@ Adjust.initSdk({
...
    environment: "production",
    logLevel: "verbose",
-   urlStrategy: "china",
+   urlStrategy: {
+      domains: ["adjust.cn", "adjust.com"],
+      useSubdomains: true,
+      isDataResidency: false,
+   },
 });
 ```
```
Comment (YaraMatkova) [inline src/content/docs/en/sdk/web/features/privacy.mdx:186]: Would it be possible to mention that `customUrl` property is also deprecated and should be replaced with `urlStartegy` with the appropriate configuration as value?

Migration example.
Old code:
```
Adjust.initSdk({
   appToken: "YOUR_APP_TOKEN",
   environment: "sandbox",
   customUrl: "https://my-custom-url.com",
});
```

New code:
```
Adjust.initSdk({
   appToken: "YOUR_APP_TOKEN",
   environment: "sandbox",
   urlStrategy: {
      domains: ["my-custom-url.com"],
      useSubdomains: false,
      isDataResidency: false,
   },
});
```

**Review by YaraMatkova — COMMENTED** [review]
Great job, thank you Ivo 😊👍

File: src/content/docs/en/sdk/web/features/privacy.mdx [hunk 5]
```diff
@@ -27,93 +27,180 @@ You can send the user's erasure request to Adjust by calling the `gdprForgetMe`
...
-<CodeBlock title="Property declaration">
+<MinorVersion
+   removed="5.7.0"
+   link="https://github.com/adjust/web_sdk/releases/tag/v5.7.0"
+/>
 
-```js
-urlStrategy: "india" | "china";
-```
+In previous versions of the SDK, you needed to pass your endpoint in the `customUrl` property of the `initSdk` method. However, it is now deprecated. To set data residency in the current SDK, use the `urlStrategy` configuration instead.
```
Comment (YaraMatkova) [inline src/content/docs/en/sdk/web/features/privacy.mdx:161]: To set _data residency_

This paragraph is about custom url, and I'm not sure if it's a typo or not 🤷‍♀️

File: src/content/docs/en/sdk/web/features/privacy.mdx [hunk 6]
```diff
@@ -27,93 +27,180 @@ You can send the user's erasure request to Adjust by calling the `gdprForgetMe`
...
 ```js
 Adjust.initSdk({
    appToken: "YOUR_APP_TOKEN",
-   environment: "sandbox",
-   customUrl: "https://my-custom-url.com/",
+   environment: "production",
+   logLevel: "verbose",
+   dataResidency: "EU",
+   urlStrategy: {
+      domains: ["adjust.cn", "adjust.com"],
```
Comment (YaraMatkova) [inline src/content/docs/en/sdk/web/features/privacy.mdx:199]: it should be `eu.adjust.com`, sorry if I pasted a wrong code 🙏

**Review by YaraMatkova — APPROVED** [review]
Thank you, the document looks very nice 👍

File: src/content/docs/en/sdk/web/features/privacy.mdx [hunk 7]
```diff
@@ -27,93 +27,180 @@ You can send the user's erasure request to Adjust by calling the `gdprForgetMe`
...
+/>
+
+The URL strategy feature allows you to set either:
+
+-  The country in which Adjust stores your data (data residency).
+-  The endpoint to which the Adjust SDK sends traffic (URL strategy).
+
+This is useful if you're operating in a country with strict privacy requirements. When you set your URL strategy, Adjust stores data in the selected data residency region or sends traffic to the chosen domain.
+
+To set your country of data residency, call the `AdjustConfig.setUrlStrategy` method following parameters:
```
Comment (ironnysh) [inline src/content/docs/en/sdk/web/features/privacy.mdx:127]: Should this be `AdjustConfig` or `initSdk`?

Comment (YaraMatkova) [reply src/content/docs/en/sdk/web/features/privacy.mdx:125]: Hi @ironnysh, it should be `initSdk`, Web SDK doesn't have `AdjustConfig.setUrlStrategy`. Thank you!

**Pull request merged**